docs/versioned-plugins/outputs/google_bigquery-v4.0.0.asciidoc

```diff
@@ -0,0 +1,349 @@
+:plugin: google_bigquery
+:type: output
+:default_codec: plain
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: %VERSION%
```

Comment:
one detail I forgot to mention, we need to fill out these variables. You can see an example here: https://raw.githubusercontent.com/elastic/logstash-docs/versioned_plugin_docs/docs/versioned-plugins/outputs/google_bigquery-v3.2.1.asciidoc